### ISSUE TITLE
请问在训练中怎么visulize训练情况么？

### ISSUE THREAD

**Issue by WaNePr** [macanv/BERT-BiLSTM-CRF-NER#69]
比方说在dev set上看到训练的情况，loss，pre/recall等？

**Comment by macanv** [comment]
目前日志冲突导致看不到loss信息。以后会修复。欢迎修改后提交pull requsest

**Comment by WaNePr** [comment]
@macanv 请问能指出日志冲突的具体位置信息么？谢谢

**Comment by macanv** [comment]
就是使用了argparser后，tf.logging.info就不打印了。

**Comment by Observerspy** [comment]
train_helper.py中用tf.app.flags替换argparser后可以正常输出日志

**Comment by tes91** [comment]
tf.logging.set_verbosity(tf.logging.INFO)

将 tf 的日志等级重新设置一下，就能显示了，看官方bert那加了logging hook

显示的样子大概是这样的：

INFO:tensorflow:loss = 220.158, step = 1
INFO:tensorflow:global_steps = 1, loss = 220.158
